package.json

```diff
@@ -14,6 +14,8 @@
     "packages/*"
   ],
   "scripts": {
+    "bootstrap": "lerna bootstrap",
```

Comment:
The `bootstrap` step was removed in #372 since my understanding was that lerna's bootstrap itself just runs `yarn install`, which will have already occurred. Does it do something else that I've missed?

Reply:
It's possible they do the same thing, `yarn` and `yarn lerna bootstrap`, but I've been using both. I'll try it out and see what happens.

Reply:
LOL you're right. I just nuked everything with `yarn lerna clean`, then `yarn` and `yarn test` and everything passes. I'll revise.